pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dto/mercury/MercuryOrderDto.java

```diff
@@ -19,14 +22,14 @@ public class MercuryOrderDto {
     String mercuryPdoId;
     String createdBy;
     int ddpInstanceId;
-    Long tissueId;
-    Long dsmKitRequestId;
+    Integer tissueId;
```

Comment:
If tissueId and dsmKitRequestId are required, consider making them int instead of Integer.